report_pre_thesis/src/sections/data_overview.tex

```diff
@@ -1,5 +1,103 @@
+    \item \texttt{shot1}, \texttt{shot2}, ..., \texttt{shot50}: The intensity measurement for each wavelength at the corresponding shot measured in photons/pulse/mm²/sr/nm.
+    \item \texttt{median}: The median of the intensity measurements for each wavelength.
+    \item \texttt{mean}: The mean of the intensity measurements for each wavelength.
+\end{itemize}
+
+\begin{table*}[!b]
+\centering
+\begin{tabular}{rrrrrrrr}
+\toprule
+     wave &         shot1 &         shot2 &  $\cdots$ &        shot49 &       shot50  & median        & mean          \\
+\midrule
+240.81100 & 6.4026649e+15 & 4.0429349e+15 & $\cdots$  & 1.7922483e+15 & 1.7126615e+15 & 1.9892956e+15 & 1.7561699e+15 \\
+240.86501 & 3.8557462e+12 & 2.2923680e+12 & $\cdots$  & 1.1355429e+12 & 8.6930379e+11 & 7.8172542e+11 & 7.2805052e+11 \\
+$\vdots$  & $\vdots$      & $\vdots$      & $\cdots$  & $\vdots$      & $\vdots$      & $\vdots$      & $\vdots$      \\
+905.38062 & 1.8823427e+08 & 58500403.     & $\cdots$  & -8449286.6    & 8710775.0     & 4.0513312e+09 & 5.2188327e+09 \\
+905.57349 & 1.9864713e+10 & 1.2956832e+10 & $\cdots$  & 1.9785415e+10 & 7.1994239e+09 & 1.1311150e+10 & 1.2201224e+10 \\
+\bottomrule
+\end{tabular}
+\caption{Example of CCS data for the first location in the \texttt{cadillac} sample directory.}
+\label{tab:ccs_data_example}
+\end{table*}
+
+The rows in the location dataset represent which wavelength the intensity measurements were taken at.
+There are $6144$ rows and $N$ columns, where $N$ is the number of shots taken for a given sample.
+While $N=50$ for each sample in the calibration data, the number of shots taken on Mars for each sample can vary but is typically between $15$ and $50$.
+
+Table \ref{tab:ccs_data_example} shows an example of the CCS data for the first location for the \texttt{cadillac} sample.
+As can be seen in the table, the second final row of the \texttt{cadillac} sample contains negative values, which is not physically possible.
+These negative values represent noise and are a result of the processing step that is applied to the EDR data to remove the background noise during transformation to RDR data, which is illustrated in Figure~\ref{fig:libs_data_processing}.
+
+Figure \ref{fig:masked_regions} shows a spectral plot of the CCS data for the \texttt{cadillac} sample.
+Note how it comprises of three different spectral regions - ultra-violet (UV), violet (VIO), and visible and near infrared (VNIR).
+For each of these regions, separate instruments are used.
+Consequently, the edges of the spectral regions are noisy because pixels at the edges of the CCD\footnote{A charge-coupled device (CCD) is a light-sensitive electronic detector that converts incoming photons into an electronic signal, commonly used in digital imaging and astronomy.\cite{radionuclide_imaging}} usually exhibit lower sensitivity compared to those at the center, and the optics vary in their reflective and absorptive properties at different wavelengths.
```

Comment:
Cite before `.`
```suggestion
Consequently, the edges of the spectral regions are noisy because pixels at the edges of the CCD\footnote{A charge-coupled device (CCD) is a light-sensitive electronic detector that converts incoming photons into an electronic signal, commonly used in digital imaging and astronomy\cite{radionuclide_imaging}.} usually exhibit lower sensitivity compared to those at the center, and the optics vary in their reflective and absorptive properties at different wavelengths.
```